# Tutorials for multimodal omics data analysis

This repository contains examples showcasing processing and interpretation of multimodal omics data using [muon](https://github.com/scverse/muon).

## Single-cell ATAC and gene expression

- [These tutorials on 10k PBMC cells](single-cell-rna-atac/pbmc10k) demonstrate how to work with chromatin accessibility and gene expression data from individual cells.

## CITE-seq

- [This tutorial on 5k PBMC cells](cite-seq/1-CITE-seq-PBMC-5k.ipynb) describes how to process CITE-seq data (including antibody-derived tags counts normalisation) and how to integrate surface protein abundance and gene expression.

## Spatiotemporal omics

- [These tutorials](mefisto/) show how smooth factors can be learnt, across time and space, with models such as [MEFISTO](https://biofam.github.io/MOFA2/MEFISTO).

## Biomedical applications

- [Chronic lymphocytic leukemia (CLL) notebook](CLL.ipynb) gives a glimpse of the utility of multi-omics data integration in biomedical applications.

# Other resources

For the list of software packages, you might find useful the [awesome multi-omics](https://github.com/mikelove/awesome-multi-omics) repository.

[//]: # (numfocus-fiscal-sponsor-attribution)

muon is part of the scverse® project ([website](https://scverse.org), [governance](https://scverse.org/about/roles)) and is fiscally sponsored by [NumFOCUS](https://numfocus.org/).
If you like scverse® and want to support our mission, please consider making a tax-deductible [donation](https://numfocus.org/donate-to-scverse) to help the project pay for developer time, professional services, travel, workshops, and a variety of other needs.

<div align="center">
<a href="https://numfocus.org/project/scverse">
  <img
    src="https://raw.githubusercontent.com/numfocus/templates/master/images/numfocus-logo.png"
    width="200"
  >
</a>
</div>
